internal/js/modules/k6/browser/common/frame.go

```diff
@@ -1002,6 +1002,58 @@ func (f *Frame) getAttribute(selector, name string, opts *FrameBaseOptions) (str
 	return s, true, nil
 }
 
+// Locator creates and returns a new locator for this frame.
```

Comment:
This GoDoc sentence looks like a leftover from copy-pasting 🤔

Reply:
Updated in f490d706646975071b6378b8e0c92399ae1ac172